src/integration/pages/storyPage/canonicalTests.js

```diff
@@ -38,4 +38,17 @@ export default () => {
       });
     }
   });
+
+  describe('Ads', () => {
+    const hasAds = service === 'mundo';
+    if (hasAds) {
+      const leaderboardEl = document.getElementById('dotcom-leaderboard');
+      const mpuEl = document.getElementById('dotcom-mpu');
+
+      it('should have ads in the document', () => {
+        expect(leaderboardEl).toBeInTheDocument();
+        expect(mpuEl).toBeInTheDocument();
```

Comment:
Thanks for adding the tests, could we have a test for a service that doesn't display ads similar to the front page: https://github.com/bbc/simorgh/blob/latest/src/integration/pages/frontPage/canonicalTests.js#L28..L43

Reply:
@andrewscfc updated.